docs/glues/herbs2rest.md

```diff
@@ -0,0 +1,96 @@
+---
+id: herbs2rest
+title: Herbs2Rest
+sidebar_label: Herbs2Rest
+slug: /glues/herbs2rest
+---
+
+Create a REST API based on herbs entities ([gotu](https://github.com/herbsjs/gotu)) and usecases ([buchu](https://github.com/herbsjs/buchu)).
+
+
+## Getting started
+### Installing
+
+    $ npm install herbs2rest
+
+### Using
+
+Use the method generateRoutes to generate api rest routes based on usecases.
+
+herbs2rest works with [express](https://expressjs.com/) in version [4.x](https://expressjs.com/en/4x/api.html).
+
+#### Controller List
+
+The method needs a list of controllers like the example below:
+
+```javascript
+const controllerList = [
+  {
+    name: 'lists',
+    idParameter: 'listId',
+    getAll: require('../usecases/getLists'),
+    getById: require('../usecases/getLists'),
+    post: require('../usecases/createList'),
+    put: require('../usecases/updateList'),
+    delete: require('../usecases/deleteList')
+  }
+]
+```
+
+The name field is the name of the route.
```

Comment:
highlight variable names in the text:
The `name` field is the name of the route.

Also, give examples: 
The `name` field is the name of the route. Ex: `https://example.com/lists/...`

Reply:
Revised